src/main/java/ai/elimu/web/project/license/LicenseCreateController.java

```diff
@@ -93,6 +94,30 @@ public String handleSubmit(
             return "project/license/create";
         } else {
             licenseDao.create(license);
+            
+            // Send license information via e-mail
```

Comment:
I would put the whole thing into a separate method. It looks big!

Reply:
Did you think of any options to put HTML code on UI side?

Reply:
> I would put the whole thing into a separate method. It looks big!

Yes, good idea. Will fix this.

Reply:
> Did you think of any options to put HTML code on UI side?

What do you mean by UI side?

Reply:
> What do you mean by UI side?

I mean JSP. Normally it's a bad practice to include html into any Java code.
But in any case, this is an optional future refactoring